mozilla/bedrock
# Ensure the slug on a particular VPN page is all lowercase

## Issue / Bugzilla link

Resolves #15386

## Testing

* Run migrations
* Confirm that http://localhost:8000/en-US/products/vpn/resource-center/no-logging-vpn-from-mozilla/ works 
* Note that http://localhost:8000/en-US/products/vpn/resource-center/no-Logging-vpn-from-mozilla/  now 404s
